package.json

```diff
@@ -62,13 +63,16 @@
     "eslint-plugin-mocha": "^3.0.0",
     "eslint-plugin-react": "^5.1.1",
     "istanbul": "^1.0.0-alpha.2",
-    "mocha": "^2.2.5",
+    "knex": "^0.11.7",
+    "mocha": "^2.5.3",
     "mocha-eslint": "^2.1.1",
+    "mongoose": "^4.5.4",
     "rollup": "^0.27.0",
     "rollup-plugin-babel": "^2.5.0",
     "should": "^6.0.3",
     "sinon": "^2.0.0-pre",
-    "sinon-chai": "https://github.com/sjmulder/sinon-chai.git#pr/sinon-2.0.0-pre"
+    "sinon-chai": "https://github.com/sjmulder/sinon-chai.git#pr/sinon-2.0.0-pre",
```

Comment:
Last I checked `sinon-chai` didn't support sinon version `2.0.0-pre` that's why the PR till the next version of `sinon-chai` is released (probably after `sinon 2.0.0`).